src/persist-client/src/internal/encoding.rs

```diff
@@ -1150,13 +1150,15 @@ impl<T: Timestamp + Codec64> RustType<ProtoU64Antichain> for Antichain<T> {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SerdeWriterEnrichedHollowBatch {
     pub(crate) shard_id: ShardId,
+    pub(crate) version: String,
```

Comment:
looks like the semver crate has a `serde` feature, which I assume adds Serialize and Deserialize impls